frontend/src/app/profile/pages/public/public.page.ts

```diff
@@ -80,4 +83,23 @@ export class PublicPage implements OnInit {
     }
 
   }
+
+  async presentToastFollow() {
```

Comment:
@didemongu should we need to create separate functions for each message we're going to make? Why not **parametrize each toast** and **utilize a service** for doing this.